### PR TITLE
 RF 00.02| New Iem > Create Pipeline Project > update testDiscardOldB…

### PR DESCRIPTION
**Pull request by SvetlanaHoshaw**
[RF 00.02| New Iem > Create Pipeline Project > testDiscardOldBuildsIsChecked](https://trello.com/c/nTG5iZr6/1697-rf-0002-new-iem-create-pipeline-project-testdiscardoldbuildsischeckedl)

### CODE REVIEW

**Comment by SvetlanaHoshaw** [comment]
[RF 00.02| New Iem > Create Pipeline Project > testDiscardOldBuildsIsChecked](https://trello.com/c/nTG5iZr6/1697-rf-0002-new-iem-create-pipeline-project-testdiscardoldbuildsischecked)

**Review by verafes — APPROVED** [review]

File: src/test/java/school/redrover/PipelineTest.java [hunk 1]
```diff
@@ -531,15 +531,13 @@ public void testSetDescription() {
...
-        getDriver().findElement(By.xpath("//*[@href='/job/test-pipeline/configure']")).click();
-
-        getDriver().findElement(By.xpath("//label[contains(text(),'Discard old builds')]")).click();
-        getDriver().findElement(By.name("Submit")).click();
-
-        getDriver().findElement(By.xpath("//*[@href='/job/test-pipeline/configure']")).click();
+        TestUtils.createPipeline(this, PIPELINE_NAME, false);
 
-        WebElement discardOldBuildsCheckbox = getDriver().findElement(By.id("cb2"));
+        WebElement discardOldBuildsCheckbox = new PipelinePage(getDriver())
```
Comment (evanmaiqa) [inline src/test/java/school/redrover/PipelineTest.java:536]: You can return a boolean and not a WebElement

Comment (SvetlanaHoshaw) [reply src/test/java/school/redrover/PipelineTest.java:536]: Should I create another pull request?

Comment (verafes) [reply src/test/java/school/redrover/PipelineTest.java:536]: You can fix the code and push a new commit to this PR

Comment (SvetlanaHoshaw) [reply src/test/java/school/redrover/PipelineTest.java:536]: Is it a suggestion for the future or I must create a new PR with changes ?

Comment (SvetlanaHoshaw) [reply src/test/java/school/redrover/PipelineTest.java]: Thank you @verafes Just did it. Fixed and pushed a new commit

Comment (verafes) [reply src/test/java/school/redrover/PipelineTest.java]: you can continue to work in your current branch, make changes in your code and push them to this PR.

**Comment by SvetlanaHoshaw** [comment]
[RF 00.02| New Iem > Create Pipeline Project > testDiscardOldBuildsIsChecked](https://trello.com/c/nTG5iZr6/1697-rf-0002-new-iem-create-pipeline-project-testdiscardoldbuildsischecked)

**Review by evanmaiqa — APPROVED** [review]

**Pull request merged**